.github/workflows/ci-release.yml

```diff
@@ -0,0 +1,66 @@
+
+name: CI
```

Comment:
maybe to keep a different name for this action? i.e., to differentiate it from the main CI with tests (`Run examples`?)

Reply:
Thanks, done.